tern/classes/file_data.py

```diff
@@ -185,6 +196,7 @@ def fill(self, file_dict):
             authors: <authors>
             packages: <packages>
             urls: <urls>
+            checksums: <checksums>
```

Comment:
Can you modify the `testFill` test to see if you can fill in a list of checksum tuples?